melodic/distribution.yaml

```diff
@@ -9625,6 +9625,21 @@ repositories:
       url: https://github.com/Terabee/teraranger_array.git
       version: master
     status: maintained
+  tf2_server:
+    doc:
+      type: git
+      url: https://github.com/peci1/tf2_server.git
```

Comment:
The source looks generally OK, but please add a LICENSE file to the repository to be clear about which BSD license is being used.  It would also follow standard practice to add the license header to every source file, but I won't insist on that.